packages/core/describers/events/lib/assetEventDescribers.js

```diff
@@ -3,90 +3,187 @@
+} from '../../../dao/constants/PlatformsManagerDAO'
+
+import {
+  TX_ISSUE,
+  TX_REVOKE,
+  TX_APPROVE,
+  TX_OWNERSHIP_CHANGE,
+  TX_RECOVERY,
+  TX_ASSET_TRANSFER,
+} from '../../../dao/constants/ChronoBankPlatformDAO'
+
+import {
+  TX_ASSET_CREATED,
+} from '../../../dao/constants/PlatformTokenExtensionGatewayManagerEmitterDAO'
+
+export const EVENT_DESC_PLATFORM_REQUESTED = new EventDescriber(
+  findEventABI(PlatformsManagerABI, EVENT_PLATFORM_REQUESTED),
+  ({ log, block }, context, { params }) => {
+
+    const { platform } = params
+
+    return new LogEventModel({
+      key: `${log.blockHash}/${log.transactionIndex}/${log.logIndex}`,
+      name: EVENT_PLATFORM_REQUESTED,
+      address: log.address,
+      date: new Date(block.timestamp * 1000),
+      icon: 'event',
+      title: `Platform ${platform} created`,
+      subTitle: platform,
+      message: params.proxy,
+      target: null,
+    })
+  },
+)
+
+export const EVENT_DESC_PLATFORM_ATTACHED = new EventDescriber(
+  findEventABI(PlatformsManagerABI, EVENT_PLATFORM_ATTACHED),
+  ({ log, block }, context, { params }) => {
+
+    const { platform } = params
+
+    return new LogEventModel({
+      key: `${log.blockHash}/${log.transactionIndex}/${log.logIndex}`,
+      name: EVENT_PLATFORM_REQUESTED,
+      address: log.address,
+      date: new Date(block.timestamp * 1000),
+      icon: 'event',
+      title: `Platform ${platform} attached`,
+      subTitle: platform,
+      message: params.proxy,
+      target: null,
+    })
+  },
+)
+
+export const EVENT_DESC_PLATFORM_DETACHED = new EventDescriber(
+  findEventABI(PlatformsManagerABI, EVENT_PLATFORM_DETACHED),
+  ({ log, block }, context, { params }) => {
+
+    const { platform } = params
+
+    return new LogEventModel({
+      key: `${log.blockHash}/${log.transactionIndex}/${log.logIndex}`,
+      name: EVENT_PLATFORM_DETACHED,
+      address: log.address,
+      date: new Date(block.timestamp * 1000),
+      icon: 'event',
+      title: `Platform ${platform} detached`,
+      subTitle: platform,
+      message: params.proxy,
+      target: null,
+    })
+  },
+)
 
 export const EVENT_ISSUE = new EventDescriber(
-  findEventABI(ChronoBankPlatformEmitterABI, 'Issue'),
+  findEventABI(ChronoBankPlatformEmitterABI, TX_ISSUE),
   ({ log, block }, context, { params }) => {
 
+    const symbol = web3Converter.bytesToString(params.symbol).toUpperCase()
+    const token = context.store.tokens.getBySymbol(symbol)
+    let value, mark
+    if (token) {
+      value = token.removeDecimals(new BigNumber(params.value))
+      mark = value > 0 ? '+' : '-'
+    } else {
+      value = 0
+    }
+
     return new LogEventModel({
       key: `${log.blockHash}/${log.transactionIndex}/${log.logIndex}`,
-      type: 'event',
-      name: 'Issue',
+      name: TX_ISSUE,
       address: log.address,
       date: new Date(block.timestamp * 1000),
       icon: 'event',
-      title: 'Issue',
+      title: `Issue ${mark}${value}`,
```

Comment:
Variable `mark` must be initialised as empty string to prevent "Issue undefined0" in 'title' property.